src/isNonNegative.js

```diff
@@ -0,0 +1,30 @@
+import { both, gte, flip } from 'ramda';
+
+import isNumber from './isNumber';
+
+/**
+ * Checks if value is a non-positive `Number` primitive or object. This includes all positive
```

Comment:
Checks if value is a non-negative `Number` primitive or object. This includes all positive numbers and zero.